apps/web/pages/signup.tsx

```diff
@@ -468,7 +468,7 @@ export default function Signup({
             </div>
           </div>
         </div>
-        <div className="border-muted mx-auto mt-24 w-full max-w-2xl flex-col justify-between rounded-l-2xl bg-[radial-gradient(162.05%_170%_at_109.58%_35%,_rgba(102,_117,_147,_0.7)_0%,_rgba(212,_212,_213,_0.4)_100%)] pl-4 dark:bg-none lg:mt-0 lg:flex lg:max-w-full lg:border lg:py-12 lg:pl-12">
+        <div className="border-muted  mx-auto mt-24 hidden w-full max-w-2xl flex-col justify-between rounded-l-2xl bg-[radial-gradient(162.05%_170%_at_109.58%_35%,_rgba(102,_117,_147,_0.7)_0%,_rgba(212,_212,_213,_0.4)_100%)] pl-4 dark:bg-none lg:mt-0 lg:block lg:flex lg:max-w-full lg:border lg:py-12 lg:pl-12">
```

Comment:
imo we should keep awards but hide features

maybe make awards a bit smaller with more spacing on mobile